test/test_workers.rb

```diff
@@ -1,3 +1,4 @@
+$LOAD_PATH.unshift(File.expand_path('../lib', __dir__))
```

Comment:
not sure how this worked before; seems like something changed with load paths I'm not aware of? require stopped working 🤔 will not investigate if the build is green tho 🙈